platforms/r3-corda-ent/charts/nmap/templates/deployment.yaml

```diff
@@ -72,40 +72,39 @@ spec:
-                   echo "Successfully got initial network parameters"
-                   break
-                fi
-                COUNTER=`expr "$COUNTER" + 1`
-              done  
-
               # Getting node Info File from Vault
-              COUNTER=1
-              while [ "$COUNTER" -lt {{ $.Values.vault.retries }} ]
-              do 
-                LOOKUP_SECRET_RESPONSE=$(curl -sS --header "X-Vault-Token: ${VAULT_TOKEN}" ${VAULT_ADDR}/v1/${CERTS_SECRET_PREFIX}/{{ $.Values.serviceLocations.notary.name }}/nodeInfo | jq -r 'if .errors then . else . end')
-                if echo ${LOOKUP_SECRET_RESPONSE} | grep "errors"
-                  then
-                    sleep {{ $.Values.vault.sleepTimeAfterError }}
-                  else
-                  notary_nodeinfo=$(echo ${LOOKUP_SECRET_RESPONSE} | jq -r '.data["nodeInfoFile"]')
-                  notary_nodeinfo_name=$(echo ${LOOKUP_SECRET_RESPONSE} | jq -r '.data["nodeInfoName"]')
-                  echo "${notary_nodeinfo}" | base64 -d > ${NODEINFO_MOUNT_PATH}/${notary_nodeinfo_name}
-                  echo "Successfully got node info file"
-                  break
-                fi
-                COUNTER=`expr "$COUNTER" + 1`
+              
+              notaries=$(echo {{ $.Values.serviceLocations.notary }} | tr -d '[]')
+              file="${NODEINFO_MOUNT_PATH}/network-parameters-initial.conf"
+              echo "notaries : [" > $file
+              for notary in $notaries
+              do
+              LOOKUP_SECRET_RESPONSE=$(curl -sS --header "X-Vault-Token: ${VAULT_TOKEN}" ${VAULT_ADDR}/v1/${CERTS_SECRET_PREFIX}/${notary}/nodeInfo | jq -r 'if .errors then . else . end')
+              if echo ${LOOKUP_SECRET_RESPONSE} | grep "errors"
+                then
+                  sleep {{ $.Values.vault.sleepTimeAfterError }}
+                else
+                notary_nodeinfo=$(echo ${LOOKUP_SECRET_RESPONSE} | jq -r '.data["nodeInfoFile"]')
+                notary_nodeinfo_name=$(echo ${LOOKUP_SECRET_RESPONSE} | jq -r '.data["nodeInfoName"]')
+                validating=$(echo ${LOOKUP_SECRET_RESPONSE} | jq -r '.data["validating"]')
+                echo "${notary_nodeinfo}" | base64 -d > ${NODEINFO_MOUNT_PATH}/${notary_nodeinfo_name}
+                echo "Successfully got node info file"
+
+                echo "  {" >> $file
+                echo "    notaryNodeInfoFile: \"notary-nodeinfo/${notary_nodeinfo_name}\"" >> $file
+                echo "    validating = \"${validating}\"" >> $file
+                echo "  }" >> $file
+
+              fi
               done
+              echo "]" >> $file
+              echo "minimumPlatformVersion = 1" >> $file
+              echo "maxMessageSize = 10485760" >> $file
+              echo "maxTransactionSize = 10485760" >> $file
+              echo "eventHorizonDays = 1" >> $file
+
+              cat $file
```

Comment:
Is this required?

Reply:
cat is not required just wanted to keep it for log checking.. will remove it